src/Tempest/Database/DatabaseFactory.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Database;
+
+use Tempest\Database\Drivers\MySqlDriver;
+use Tempest\Database\Drivers\PostgreSqlDriver;
+use Tempest\Database\Drivers\SQLiteDriver;
+
+final class DatabaseFactory
+{
+    public static function make(DatabaseDialect $dialect, array $options): DatabaseConfig
```

Comment:
Should return `DatabaseDriver`.

Reply:
I just read your message in Discord again and was like wait a minute 😋